torch/tensor.py

```diff
@@ -425,8 +425,13 @@ def __contains__(self, element):
         """
         if isinstance(element, (torch.Tensor, Number)):
             return (element == self).any().item()
-        return NotImplemented
 
+        raise RuntimeError(
+            "Tensor.__contains__ only supports scalars, but you passed in a %s. "
```

Comment:
this message doesn't appear correct -- directly above it says that we support tensors, and we certainly do:
```
>>> torch.tensor([1,2,3]) in torch.tensor([1,2,3])
True
```

Reply:
Yeah, thanks for your advice. I think we should follow the doc:
```
"""Check if `element` is present in tensor

        Arguments:
            element (Tensor or scalar): element to be checked
                for presence in current tensor"
"""
```
And modify the error message to:
```
Tensor.__contains__ only supports Tensor and scalar ...
```